.github/workflows/ci-angular.yaml

```diff
@@ -0,0 +1,80 @@
+name: Angular CI
+
+on:
+  push:
+    branches: [ "master" ]
+env:
+  IMAGE_NAME: angular
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
GH_TOKEN

There is no value in just renaming default variable names, other engineers are used to. 

```
GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

so just use secrets.GITHUB_TOKEN where you need it.

Reply:
Fixed

Reply:
you can use `secrets.GITHUB_TOKEN` anywhere in the steps as it's always available. No need for a custom ENV var that is just an alias to `secrets.GITHUB_TOKEN`